engine/verification/fetcher/chunkconsumer/jobs.go

```diff
@@ -0,0 +1,25 @@
+package chunkconsumer
```

Comment:
The following logic is **not new**, we just breakdown a longer file into smaller ones each representing a separate logic on this package.